renku/core/models/workflow/converters/__init__.py

```diff
@@ -16,3 +16,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Define classes for converting workflows to workflow languages."""
+
+from __future__ import annotations
+
+from abc import ABCMeta, abstractmethod
+from pathlib import Path
+from typing import List, Optional, Tuple, Union
+
+from renku.core.models.workflow.composite_plan import CompositePlan
+from renku.core.models.workflow.plan import Plan
+
+
+class WorkflowConverter(metaclass=ABCMeta):
```

Comment:
Not that we really discussed this convention (I jsut added it in my PR), but I think it makes sense to have interfaces start with I (`IWorkflowConverter`) so it's clear they're interfaces (Since python doesn't support proper interfaces).